### ISSUE TITLE
ValidityState broken link to customError

### ISSUE THREAD

**Issue by qw-in** [mdn/content#26522]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/ValidityState

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/API/ValidityState#browser_compatibility

### What information was incorrect, unhelpful, or incomplete?

The linked to `customError` page is missing: https://developer.mozilla.org/en-US/docs/Web/API/ValidityState/customError

### What did you expect to see?

Something similar to the page for `badInput`: https://developer.mozilla.org/en-US/docs/Web/API/ValidityState/badInput

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/validitystate`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/ValidityState
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/validitystate/index.md
* Last commit: https://github.com/mdn/content/commit/6b1e3eebf22abf1b73bb219581335b1147b75d7a
* Document last modified: 2023-03-12T09:29:44.000Z

</details>

**Comment by hamishwillee** [comment]
Yes, the compatibility data is expecting the page to exist, but it has not been created yet. Would you like to do so?

**Comment by teoli2003** [comment]
I've created a PR to fix this problem last week. I need to finish it so that it lands: mdn/content#26431
